ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2

```diff
@@ -39,11 +39,7 @@ daemon TerminAttr
 {%         else %}
 {%             set cvp_config.cli = cvp_config.cli ~ " -ingestexclude=" %}
 {%         endif %}
-{%         if daemon_terminattr.ingestvrf is arista.avd.defined %}
-{%             if daemon_terminattr.ingestvrf != 'default' %}
-{%                 set cvp_config.cli = cvp_config.cli ~ " -ingestvrf=" ~ daemon_terminattr.ingestvrf %}
-{%             endif %}
-{%         endif %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -ingestvrf=" ~ daemon_terminattr.ingestvrf | default('default') %}
```

Comment:
When I raised this in #950 there was no objection.  As I stated when CVP onboards a device, if the default vrf is used it uses the config synxtax:

```
daemon TerminAttr
    exec /usr/bin/TerminAttr -ingestgrpcurl=10.10.10.10:9910 -cvcompression=gzip -ingestauth=key -smashexcludes=test -ingestexclude=test -ingestvrf default -taillogs
```

If you want me to revert this change, there is no point for this PR.

Reply:
CVP produces config with and without the ingestvrf depending on if you onboard existing device, zero-touch a device or use the system configletbuilder. So it is difficult to say that one is more correct than the other.
But even if ingestvrf is desired, the role eos_cli_config_gen should not be the enforcing any "good practices". It is just a translator. Any abstraction and logic should go into eos_designs.
Re. #950 there is no mention of always setting default VRF if not VRF is given. So my objection is not about #950 but only this choice of always setting the VRF.
If we change default config output like this, it will affect all users of AVD.

Reply:
Fair enough.  Your point, which I misunderstood is that I should allow no vrf to be set and if no vrf to be set, not to define a default value for vrf if not set.